sw/device/lib/crypto/impl/kdf.c

```diff
@@ -80,8 +102,10 @@ otcrypto_status_t otcrypto_kdf_kmac(
 
   // Check `keying_material` key length.
   if (keying_material->config.hw_backed == kHardenedBoolTrue) {
+    // The case where `keying_material` is hw_backed is addressed by
+    // `otcrypto_hw_backed_key` function in `key_transport.h`.
     return OTCRYPTO_NOT_IMPLEMENTED;
```

Comment:
I'd use `OTCRYPTO_BAD_ARGS` here; `OTCRYPTO_NOT_IMPLEMENTED` is for things we plan to implement one day but haven't yet, while I think this one we never intend to support.